src/main/java/net/minecraftforge/client/ForgeRenderTypes.java

```diff
@@ -206,5 +230,43 @@ public static RenderType layeredItemTranslucent(ResourceLocation locationIn) {
                     .build(true);
             return makeType("forge_item_entity_translucent_cull", DefaultVertexFormats.ENTITY, 7, 256, true, true, rendertype$state);
         }
+
+        public static RenderType getText(ResourceLocation locationIn) {
+            RenderType.State rendertype$state = RenderType.State.getBuilder()
+                    .texture(new CustomizableTextureState(locationIn, () -> ForgeRenderTypes.enableTextTextureLinearFiltering, () -> false))
+                    .alpha(DEFAULT_ALPHA)
+                    .transparency(TRANSLUCENT_TRANSPARENCY)
+                    .lightmap(LIGHTMAP_ENABLED)
+                    .build(false);
+            return makeType("forge_text", DefaultVertexFormats.POSITION_COLOR_TEX_LIGHTMAP, 7, 256, false, true, rendertype$state);
```

Comment:
Not exactly sure if should keep the vanilla name or add the "forge_" prefix

Reply:
Forge prefix is good.

Reply:
It's replacing the vanilla ones, so I was like it should have the same key

Reply:
Yeah that is a valid point. But this works as well. Especially since the rendering types are more or less named instances. And this makes it clear that it is not the original text rendering type. But an adapted one by forge.